config/manifests/bases/clusterserviceversion.yaml.in

```diff
@@ -3,7 +3,7 @@ kind: ClusterServiceVersion
 metadata:
   annotations:
     alm-examples: '[]'
-    capabilities: Basic Install
+    capabilities: Seamless Upgrades
```

Comment:
can you please update the PR description about why we are doing this change and point to https://sdk.operatorframework.io/docs/overview/operator-capabilities/?